KrakenIoc/KrakenIoc/Core/V1/Injector.cs

```diff
@@ -473,7 +473,8 @@ public void Inject(object objValue, IInjectContext parentContext)
 
                     object propertyValue = _container.ResolveWithCategory(property.PropertyType, memberCache.InjectAttribute.Category, injectContext);
                     
-                    property.SetValue(objValue, propertyValue, null);
+                    PropertyInfo strongProperty = property.DeclaringType.GetProperty(property.Name, BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
This code negates the caching that is built in. `property` is cached, therefore performant at large request loads. However, this code addition is not cached. Instead, make these changes to the caching of the specific property.

Reply:
That makes sense. I took the the least invasive approach above. Will update this.